drms/__main__.py

```diff
@@ -1,24 +0,0 @@
-from __future__ import absolute_import, division, print_function
```

Comment:
I removed this.

Reply:
The "CLI" works and I really like it as an easy way to get an interactive drms client up and running, e.g. by using

    ipython -i -m drms -- --email=name@example.com

to set an email for export requests, or

    ipython -i -m drms -- kis

for using a server different than JSOC.

Does it do any harm to just keep it?

Reply:
Since the CLI is obviously useful, should we maybe refactor it rather than removing it? Maybe using something like the `click` package and providing a more proper entrypoint?

Reply:
I will revert the change.

Reply:
It will need some tests and docs in future.

Reply:
I'm not sure about using `click`, but the "CLI" could definitely be improved. :)

Originally I planned to add an `idrms` command that could be used to start an interactive, "drms-enabled" IPython session (like `isympy` does for `sympy`). But I never added a binary, because

    ipython -i -m drms

just worked fine.

Reply:
I have restored in "main.py" and added some basic tests. The docs can come another time.